scripts/serving/wrapper/preprocessor.py

```diff
@@ -13,6 +13,7 @@
 SKLEARN_MODEL = "model.joblib"
 XGBOOST_MODEL = "model.xgboost"
 
+# FIXME: why do we even need this class?
```

Comment:
What's meaning? We can discuss it， if don‘t need it， we can create a issue to fix it， not  suggest comment in code

Reply:
make sense

Reply:
I'll remove these comments once finished, please note this PR is still work in progress, thanks for noticing